src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const generalCSQuiz = [
+  {
+    Question: "What does NFC stand for?",
+    Answer: "Near-Field Communication",
+    Distractor1: "Narrow Field Communication",
+    Distractor2: "Near-Fragile Communication",
+    Distractor3: "None of the above",
```

Comment:
remember that the quiz answers can show up in any order. I would suggest a different answer here

```suggestion
    Distractor3: "Nearly Factory Communication",
```